webapp/channels/src/components/widgets/menu/menu_items/menu_item.tsx

```diff
@@ -34,7 +34,7 @@ export default function menuItem(Component: React.ComponentType<any>) {
                 textProp = (
                     <>
                         <span className='icon'>{icon}</span>
-                        {text}
+                        <div className='text'>{text}</div>
```

Comment:
This only wraps the text if there is an icon. If there is no icon, it would still fail to show the ellipsis, since the style is only applied to the div.

Is that intended?

Reply:
I removed the icon from code its still showing ellipsis.